viper/modules/rats/jrat.py

```diff
@@ -158,13 +169,14 @@ def parse_config(raw_config, enckey):
     config_dict["EncryptionKey"] = enckey.encode('hex')
     return config_dict
 
+
 def config(data):
     enckey, conf = get_parts(data)
-    if enckey == None:
+    if not enckey:
```

Comment:
Did you make sure of the comportment if enckey is 0, '' or []? `if enckey is None` might be preferred there.